app/models/reaction.rb

```diff
@@ -4,7 +4,7 @@ class Reaction < ApplicationRecord
   SEARCH_SERIALIZER = Search::ReactionSerializer
   SEARCH_CLASS = Search::Reaction
 
-  CATEGORIES = %w[like readinglist unicorn thinking hands thumbsdown vomit].freeze
+  CATEGORIES = %w[like readinglist unicorn thinking hands thumbsup thumbsdown vomit].freeze
```

Comment:
this is an example of why you don't want to have enums in the DB after all @atsmith813  :D 
Our `reactions` table is probably one of the biggest ones and it would have required a complicated dance of `ALTER TABLE` with such a massive table